diff -Naur USD-21.08-original/cmake/defaults/Packages.cmake USD-21.08/cmake/defaults/Packages.cmake
--- USD-21.08-original/cmake/defaults/Packages.cmake	2021-07-15 02:00:41.000000000 -0400
+++ USD-21.08/cmake/defaults/Packages.cmake	2021-08-10 10:41:05.317624337 -0400
@@ -188,6 +188,7 @@
     # --OpenImageIO
     if (PXR_BUILD_OPENIMAGEIO_PLUGIN)
         find_package(OpenEXR REQUIRED)
+	find_package(Imath REQUIRED)
         find_package(OpenImageIO REQUIRED)
         add_definitions(-DPXR_OIIO_PLUGIN_ENABLED)
     endif()
@@ -254,7 +255,7 @@
     endif()
     # --OpenVDB
     if (PXR_ENABLE_OPENVDB_SUPPORT)
-        find_package(OpenEXR REQUIRED)
+        find_package(Imath REQUIRED)
         find_package(OpenVDB REQUIRED)
         add_definitions(-DPXR_OPENVDB_SUPPORT_ENABLED)
     endif()
@@ -283,7 +284,7 @@
 
 if (PXR_BUILD_ALEMBIC_PLUGIN)
     find_package(Alembic REQUIRED)
-    find_package(OpenEXR REQUIRED)
+    find_package(Imath REQUIRED)
     if (PXR_ENABLE_HDF5_SUPPORT)
         find_package(HDF5 REQUIRED
             COMPONENTS
@@ -305,6 +306,7 @@
 if(PXR_ENABLE_OSL_SUPPORT)
     find_package(OSL REQUIRED)
     find_package(OpenEXR REQUIRED)
+    find_package(Imath REQUIRED)
     add_definitions(-DPXR_OSL_SUPPORT_ENABLED)
 endif()
 
diff -Naur USD-21.08-original/cmake/modules/FindOpenEXR.cmake USD-21.08/cmake/modules/FindOpenEXR.cmake
--- USD-21.08-original/cmake/modules/FindOpenEXR.cmake	2021-07-15 02:00:41.000000000 -0400
+++ USD-21.08/cmake/modules/FindOpenEXR.cmake	1969-12-31 19:00:00.000000000 -0500
@@ -1,101 +0,0 @@
-#
-# Copyright 2016 Pixar
-#
-# Licensed under the Apache License, Version 2.0 (the "Apache License")
-# with the following modification; you may not use this file except in
-# compliance with the Apache License and the following modification to it:
-# Section 6. Trademarks. is deleted and replaced with:
-#
-# 6. Trademarks. This License does not grant permission to use the trade
-#    names, trademarks, service marks, or product names of the Licensor
-#    and its affiliates, except as required to comply with Section 4(c) of
-#    the License and to reproduce the content of the NOTICE file.
-#
-# You may obtain a copy of the Apache License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the Apache License with the above modification is
-# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-# KIND, either express or implied. See the Apache License for the specific
-# language governing permissions and limitations under the Apache License.
-#
-
-find_path(OPENEXR_INCLUDE_DIR
-    OpenEXR/half.h
-HINTS
-    "${OPENEXR_LOCATION}"
-    "$ENV{OPENEXR_LOCATION}"
-PATH_SUFFIXES
-    include/
-DOC
-    "OpenEXR headers path"
-)
-
-if(OPENEXR_INCLUDE_DIR)
-  set(openexr_config_file "${OPENEXR_INCLUDE_DIR}/OpenEXR/OpenEXRConfig.h")
-  if(EXISTS ${openexr_config_file})
-      file(STRINGS
-           ${openexr_config_file}
-           TMP
-           REGEX "#define OPENEXR_VERSION_STRING.*$")
-      string(REGEX MATCHALL "[0-9.]+" OPENEXR_VERSION ${TMP})
-
-      file(STRINGS
-           ${openexr_config_file}
-           TMP
-           REGEX "#define OPENEXR_VERSION_MAJOR.*$")
-      string(REGEX MATCHALL "[0-9]" OPENEXR_MAJOR_VERSION ${TMP})
-
-      file(STRINGS
-           ${openexr_config_file}
-           TMP
-           REGEX "#define OPENEXR_VERSION_MINOR.*$")
-      string(REGEX MATCHALL "[0-9]" OPENEXR_MINOR_VERSION ${TMP})
-  endif()
-endif()
-
-foreach(OPENEXR_LIB
-    Half
-    Iex
-    Imath
-    IlmImf
-    IlmThread
-    IlmImfUtil
-    IexMath
-    )
-
-    # OpenEXR libraries may be suffixed with the version number, so we search
-    # using both versioned and unversioned names.
-    find_library(OPENEXR_${OPENEXR_LIB}_LIBRARY
-        NAMES
-            ${OPENEXR_LIB}-${OPENEXR_MAJOR_VERSION}_${OPENEXR_MINOR_VERSION}
-            ${OPENEXR_LIB}
-        HINTS
-            "${OPENEXR_LOCATION}"
-            "$ENV{OPENEXR_LOCATION}"
-        PATH_SUFFIXES
-            lib/
-        DOC
-            "OPENEXR's ${OPENEXR_LIB} library path"
-    )
-
-    if(OPENEXR_${OPENEXR_LIB}_LIBRARY)
-        list(APPEND OPENEXR_LIBRARIES ${OPENEXR_${OPENEXR_LIB}_LIBRARY})
-    endif()
-endforeach(OPENEXR_LIB)
-
-# So #include <half.h> works
-list(APPEND OPENEXR_INCLUDE_DIRS ${OPENEXR_INCLUDE_DIR})
-list(APPEND OPENEXR_INCLUDE_DIRS ${OPENEXR_INCLUDE_DIR}/OpenEXR)
-
-include(FindPackageHandleStandardArgs)
-find_package_handle_standard_args(OpenEXR
-    REQUIRED_VARS
-        OPENEXR_INCLUDE_DIRS
-        OPENEXR_LIBRARIES
-    VERSION_VAR
-        OPENEXR_VERSION
-)
-
diff -Naur USD-21.08-original/pxr/imaging/hioOpenVDB/CMakeLists.txt USD-21.08/pxr/imaging/hioOpenVDB/CMakeLists.txt
--- USD-21.08-original/pxr/imaging/hioOpenVDB/CMakeLists.txt	2021-07-15 02:00:41.000000000 -0400
+++ USD-21.08/pxr/imaging/hioOpenVDB/CMakeLists.txt	2021-08-10 10:39:27.478207730 -0400
@@ -14,7 +14,7 @@
         hio
         tf
         usd
-        ${OPENEXR_Half_LIBRARY}
+	${IMATH_LIBRARIES}
         ${OPENVDB_LIBRARY}
 
     INCLUDE_DIRS
diff -Naur USD-21.08-original/pxr/imaging/plugin/hioOiio/CMakeLists.txt USD-21.08/pxr/imaging/plugin/hioOiio/CMakeLists.txt
--- USD-21.08-original/pxr/imaging/plugin/hioOiio/CMakeLists.txt	2021-07-15 02:00:41.000000000 -0400
+++ USD-21.08/pxr/imaging/plugin/hioOiio/CMakeLists.txt	2021-08-10 10:39:31.578183423 -0400
@@ -17,6 +17,7 @@
         tf
         ${OIIO_LIBRARIES}
         ${OPENEXR_LIBRARIES}
+	${IMATH_LIBRARIES}
 
     INCLUDE_DIRS
         ${OIIO_INCLUDE_DIRS}
diff -Naur USD-21.08-original/pxr/usd/plugin/usdAbc/CMakeLists.txt USD-21.08/pxr/usd/plugin/usdAbc/CMakeLists.txt
--- USD-21.08-original/pxr/usd/plugin/usdAbc/CMakeLists.txt	2021-07-15 02:00:41.000000000 -0400
+++ USD-21.08/pxr/usd/plugin/usdAbc/CMakeLists.txt	2021-08-10 11:55:33.840880041 -0400
@@ -23,15 +23,12 @@
         usd
         usdGeom
         ${ALEMBIC_LIBRARIES}
-        ${OPENEXR_Half_LIBRARY} 
-        ${OPENEXR_Imath_LIBRARY} 
-        ${OPENEXR_Iex_LIBRARY} 
-        ${OPENEXR_IexMath_LIBRARY}
+	${IMATH_LIBRARIES}
         ${optionalLibs}
 
     INCLUDE_DIRS
         ${ALEMBIC_INCLUDE_DIR}
-        ${OPENEXR_INCLUDE_DIRS}
+	${IMATH_INCLUDE_DIRS}
         ${optionalIncludeDirs}
 
     PRIVATE_CLASSES
