diff -Naur USD-21.11-original/pxr/base/arch/mallocHook.cpp USD-21.11/pxr/base/arch/mallocHook.cpp
--- USD-21.11-original/pxr/base/arch/mallocHook.cpp	2021-11-01 13:10:51.000000000 -0400
+++ USD-21.11/pxr/base/arch/mallocHook.cpp	2022-03-31 17:26:22.176739670 -0400
@@ -177,7 +177,7 @@
        _underlyingMemalignFunc || _underlyingFreeFunc;
 }
 
-#if defined(ARCH_OS_LINUX)
+#if 0
 template <typename T>
 static bool _GetSymbol(T* addr, const char* name, string* errMsg) {
     if (void* symbol = dlsym(RTLD_DEFAULT, name)) {
@@ -241,8 +241,8 @@
     ARCH_UNUSED_ARG void  (*freeWrapper)(void*, const void*),
     string* errMsg)
 {
-#if !defined(ARCH_OS_LINUX)
-    *errMsg = "ArchMallocHook functionality not implemented for non-linux systems";
+#if 1
+    *errMsg = "ArchMallocHook functionality not implemented";
     return false;
 #else
     if (IsInitialized()) {
